File: solutions/javascript/pangram/1/pangram.js
//
// This is only a SKELETON file for the 'Pangram' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const isPangram = (input) => {
  const alphabet = 'abcdefghijklmnopqrstuvwxyz';


 for (let i = 0; i < alphabet.length; i++) {

  if(!input.toLowerCase().includes(alphabet[i])){
    return false;
  }
}
return true;
};

File: solutions/csharp/pangram/1/Pangram.cs
using System;

public static class Pangram
{
    public static bool IsPangram(string input)
    {
        //throw new NotImplementedException();
        char[] alphabet =  { 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z' };


        foreach (char letter in alphabet)
        {
            if (!input.ToLower().Contains(letter)) return false;

        }
        return true;
    }
}
